packages/web/src/components/cover-photo/CoverPhoto.tsx

```diff
@@ -41,13 +42,18 @@ const CoverPhoto = ({
   error,
   edit = false,
   darken = false,
+  onClick = () => {},
   onDrop
 }: CoverPhotoProps) => {
   const [processing, setProcessing] = useState(false)
   const gradient = darken
     ? 'linear-gradient(180deg, rgba(0,0,0,0.25) 0%, rgba(0,0,0,0.75) 100%)'
     : 'linear-gradient(rgba(0, 0, 0, 0) 0%, rgba(0, 0, 0, 0.05) 70%, rgba(0, 0, 0, 0.2) 100%)'
 
+  const hasCoverPhoto = Boolean(
```

Comment:
Feel like there is a better way to do this, `hasProfilePicture` is computed like this:

```ts
    const hasProfilePicture = profile
      ? !!profile.profile_picture ||
        !!profile.profile_picture_sizes ||
        updatedProfilePicture
      : false
```
Wonder if something similar would work?